src/js/eventHandler.spec.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { callbackOnFormSubmit, callbackWhenEnterIsPressed } from './eventHandlers';
+
+document.body.innerHTML = `
+<form class="chat-window__sender-features">
+  <label for="message-area" class="visually-hidden">Message area</label>
+  <textarea
+    name="message-area"
+    id="message-area"
+    class="chat-window__input">Hello World</textarea>
+  <button class="chat-window__send-button" type="submit">Send</button>
+</form>
+`;
+
+describe('eventHandler.js', () => {
+  it('Should run callback and set textarea to empty string on "Enter" key press', () => {
```

Comment:
It would be nice to split this into 2 separate tests.  One for calling the `callback` function and one for resetting the text. That way each test is only concerned with one thing.